src/azure-cli/devcontainer-feature.json

```diff
@@ -12,16 +12,21 @@
       ],
       "default": "latest",
       "description": "Select or enter an Azure CLI version. (Available versions may vary by Linux distribution.)"
+    },
+    "extensions": {
+      "type": "array",
```

Comment:
We currently don't have an `array` type for options (although I agree it's a good idea).  

This could be a "string" type space or comma delimited.  See [nix](https://github.com/devcontainers/features/blob/3020d6da64798ee994d407ffd3ce3f800f6dff92/src/nix/devcontainer-feature.json#L22) for an example.

Reply:
Thanks for the example. The type has been switched to "string"